retrieve.py

```diff
@@ -49,81 +51,87 @@ def userInputJson()-> tuple[str, str, str, dict, list[str]]:
     print("\n")
 
     if filter == "y":
-      location  = get_Location(input("What location(city) would you like to eat at? [include country and state if you can]"))
+      location  = get_location(input("What location(city) would you like to eat at? [include country and state if you can]"))
       print("\nApplicable cuisine List: ")
       print(applicable_cuisine_list)
       print("\n")
       cuisine = input("What cuisine would you like to eat? format:\nJapanese, Korean, American").lower()
       cuisines_list = [cuisine.strip() for cuisine in cuisine.split(',')]
+      #check if user inputted any cuisines
       if cuisines_list[0] =='':
         cuisines_list = []
 
     return date, party_size, time, location, cuisines_list
 
-def get_Location(address:str) -> dict:
-  gn = geocoders.GeoNames(username='yunjun505')
 
+#Take in String input of location and return dictionary of the location's latitude and longitude
+def get_location(address:str) -> dict:
+  gn = geocoders.GeoNames(username='yunjun505')
   try:
       location = gn.geocode(address)
-
       if location:
           return {"latitude":location.latitude,"longitude":location.longitude,"radius":35420}
       else:
           print("Address not found")
           return {"latitude":location.latitude,"longitude":location.longitude,"radius":35420}
-
   except Exception as e:
       print("Error, defaulting to NYC")
       return {"latitude":location.latitude,"longitude":location.longitude,"radius":35420}
   
-def getRestaurants(date: str = formatted_date, party_size: str = "2", time: str = "", location: dict = {"latitude":40.712941,"longitude":-74.006393,"radius":35420}, cuisine_list: list[str] = []) -> list[dict]:
+
+# return restaurant list based on the user's inputs through querying and call filtered_restaurants method to filter for the cuisine 
+def get_restaurants(date: str = formatted_date, party_size: str = "2", time: str = "", location: dict = {"latitude":40.712941,"longitude":-74.006393,"radius":35420}, cuisine_list: list[str] = []) -> list[dict]:
```

Comment:
clean up this function header, for example can add the default location for NYC as a const (since we use it here and also in the prev function), also multiline function header

```
def get_restaurants(
    date : str = formatted date,
    party_size: str = "2",
    etc, etc
) - > list[dict]:
```